test/util/TestWrapper.js

```diff
@@ -162,6 +174,22 @@ class TestWrapper {
         });
     }
 
+    setCookies(res, cookies = []) {
+        if (cookies.length) {
+            let cookiesString;
+
+            for (let i=0; i<cookies.length; i++) {
+                const cookieName = cookies[i].name;
+                const cookieContent = JSON.stringify(cookies[i].content);
+                cookiesString = `${cookieName}=${cookieContent},`;
+            }
+
+            cookiesString = cookiesString.substring(0, cookiesString.length - 1);
+
+            res.set('Cookie', cookiesString);
+        }
+    }
```

Comment:
It would be better for this method to return the cookie string and then to set that to `res` in the calling method.

This way `setCookies()` has predictable input and outputs.